modules/openapi-generator/src/main/resources/cpp-qt5-client/api-header.mustache

```diff
@@ -18,7 +18,7 @@ class {{classname}}: public QObject {
 
 public:
     {{classname}}();
-    {{classname}}(QString host, QString basePath);
+    {{classname}}(const QString& host, const QString& basePath);
```

Comment:
```suggestion
    {{classname}}(const QString& host, const QString& basePath = "{{{basePathWithoutHost}}}");
```

Reply:
Yes good idea.

Reply:
I had to revert this when I provide default values, because then it looks like the default constructor with no parameters.